src/WorkOS.net/Services/UserManagement/Enums/Type.cs

```diff
@@ -0,0 +1,16 @@
+namespace WorkOS
+{
+    using System.Runtime.Serialization;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+
+    /// <summary>
+    /// An enum describing the type of reason for unauthorized access to an organization. <see cref="User"/>.
+    /// </summary>
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum Type
```

Comment:
Could we give this a more descriptive name?

```suggestion
    public enum SessionUnauthorizedOrganizationReason
```

Reply:
Would we want to rename this field from `type` across the board then?
<img width="382" alt="CleanShot 2023-08-18 at 12 15 21@2x" src="https://github.com/workos/workos-dotnet/assets/5375567/97d33d0b-51f5-4c8a-a09f-67fe9b7574ce">

Reply:
No, the field name should still be `Type`, it is just the enum name that should be changed.